ocrd_anybaseocr/mrcnn/model.py

```diff
@@ -411,7 +411,7 @@ def call(self, inputs):
             # Keep track of which box is mapped to which level
             box_to_level.append(ix)
 
-            # Stop gradient propogation to ROI proposals
+            # Stop gradient propagation to ROI proposals
```

Comment:
Bad idea to start patching the mrcnn subpackage (and for no real reason).

Background: When DFKI created this module, they took https://github.com/matterport/Mask_RCNN as a base for the block segmenter. But they did not add it as a runtime dependency, but instead decided to use the code directly – without any git history and without even mentioning which version they took, just by c&p. That's obviously a very bad choice, but one we have not had the chance to correct yet. (One would first need to work out which version they took, and then re-apply their own patches.) Part of the problem is that upstream is not maintained since 2018.

Reply:
We already patched that file, and fixing typos to silence `codespell` is a reason.

Reply:
I don't use codespell, and if I was, I wouldn't activate it for that subpackage for the reason I just gave. By `good reason` I meant a technical reason (which is what we did have to patch before).